lib/confluent_schema/server.ex

```diff
@@ -37,6 +38,25 @@ defmodule ConfluentSchema.Server do
 
   @doc false
   @spec handle_info(atom(), map()) :: {:noreply, map()}
+  def handle_info(:cache, state = %{local: true, app_name: app_name}) do
+    schema_dir =
+      app_name
+      |> :code.priv_dir()
+      |> Path.join("confluent_schema/")
+
+    File.mkdir_p!(schema_dir)
+
+    schema_dir
+    |> Path.join("*.json")
```

Comment:
Would it be helpful to have different files for staging/prod?

Reply:
No, staging/production shouldn't use `local` mode. They fetch the schemas from http://confluent.cloud

Reply:
We didn't pay for enforcing schema on cloud, but we can still fetch schema from them right?